packages/fluentui/digest/package.json

```diff
@@ -19,8 +19,7 @@
   },
   "dependencies": {
     "just-scripts": "1.3.1",
-    "querystring": "^0.2.0",
-    "webpack": "5.21.2"
```

Comment:
This looks redundant as we have the same version on root...

Reply:
It might be there because `digest` was intended to be published at some point? Though looks like it's not published now, so agreed it's not needed.